bears/vcs/git/GitCommitBear.py

```diff
@@ -34,14 +34,22 @@ class GitCommitBear(GlobalBear):
             'issue': r'(?:\w+/\w+)?#(\S+)',
             'full issue': r'https?://gitlab\S+/issues/(\S+)',
         },
+        'bitbucket': {
+            'issue': r'#(\S+)',
+            'full issue': None,
+        },
     }
     SUPPORTED_HOST_KEYWORD_REGEX = {
-        'github': (r'[Cc]lose[sd]?'
+        'github': (r'(?:[Cc]lose[sd]?'
                    r'|[Rr]esolve[sd]?'
-                   r'|[Ff]ix(?:e[sd])?'),
-        'gitlab': (r'[Cc]los(?:e[sd]?|ing)'
+                   r'|[Ff]ix(?:e[sd])?)\s+'),
+        'gitlab': (r'(?:[Cc]los(?:e[sd]?|ing)'
                    r'|[Rr]esolv(?:e[sd]?|ing)'
-                   r'|[Ff]ix(?:e[sd]|ing)?')
+                   r'|[Ff]ix(?:e[sd]|ing)?)\s+'),
+        'bitbucket': (r'(?:(?:[Cc]los(?:e[sd]?|ing)'
+                      r'|[Rr]esolv(?:e[sd]?|ing)'
+                      r'|[Ff]ix(?:e[sd]|ing))'
+                      r'(?:\s+(?:(?:bug|issue|ticket)[ \t]*)?))'),
```

Comment:
The ```\s+``` in ```r'(?:\s+(?:(?:bug|issue|ticket)[ \t]*)?))'),``` should rather be ```[ \t]+``` as an individual reference needs to be a single line.